### PR TITLE
CIV-10494 Edit judgment

### PR DESCRIPTION
**Pull request by bhagyashreesharma90**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/CIV-10494

### Change description ###
Screenshot_6-3-2024_152256_localhost
Screenshot_6-3-2024_152424_localhost

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

**Review by hmcts-version1-ignacio — APPROVED** [review]

**Review by Omaira-Melo-Hmcts — DISMISSED** [review]

File: e2e/tests/api_tests/judgmentOnline/api_judgment_online_spec_1v1_test.js [hunk 1]
```diff
@@ -3,6 +3,7 @@
 const config = require('../../../config.js');
 const mpScenario = 'ONE_V_ONE';
 const caseWorkerUser = config.hearingCenterAdminWithRegionId1;
```
Comment (Azam-Hmcts) [inline e2e/tests/api_tests/judgmentOnline/api_judgment_online_spec_1v1_test.js:5]: I would suggest using user for "caseWorkerUser" , "JudgeUser" if EA courts are enabled or not

File: e2e/tests/api_tests/judgmentOnline/api_judgment_online_spec_1v2_test.js [hunk 2]
```diff
@@ -3,13 +3,14 @@
 const config = require('../../../config.js');
 const mpScenario = 'ONE_V_TWO';
 const caseWorkerUser = config.hearingCenterAdminWithRegionId1;
```
Comment (Azam-Hmcts) [inline e2e/tests/api_tests/judgmentOnline/api_judgment_online_spec_1v2_test.js:5]: I would suggest using user for "caseWorkerUser" , "JudgeUser" if EA courts are enabled or not

**Review by ShwetaTandel-hmcts — APPROVED** [review]

**Review by hmcts-version1-ignacio — DISMISSED** [review]

File: e2e/tests/api_tests/judgmentOnline/api_judgment_online_spec_1v2_test.js (hunk 2 above)
Comment (bhagyashreesharma90) [reply e2e/tests/api_tests/judgmentOnline/api_judgment_online_spec_1v2_test.js]: Done

File: e2e/tests/api_tests/judgmentOnline/api_judgment_online_spec_1v1_test.js (hunk 1 above)
Comment (bhagyashreesharma90) [reply e2e/tests/api_tests/judgmentOnline/api_judgment_online_spec_1v1_test.js]: Done

**Review by Azam-Hmcts — APPROVED** [review]

**Review by hmcts-version1-ignacio — APPROVED** [review]

**Review by Omaira-Melo-Hmcts — APPROVED** [review]

**Pull request merged**